tests/integration_tests/test_autopilot_training_udf_real.py

```diff
@@ -63,16 +64,36 @@ def get_emitted(self):
 @pytest.mark.skipif(not aws_params.aws_access_key,
                     reason="AWS credentials are not set")
 def test_autopilot_training_udf_real():
+    problem_types_dict = {
+        'regression': {
+            'setup_params': reg_setup_params,
+            'problem_params': {
+                "problem_type": "Regression",
+                "objective": '{"MetricName":"MSE"}'
+            }},
+        'classification': {
+            'setup_params': cls_setup_params,
+            'problem_params': {
+                "problem_type": "BinaryClassification",
+                "objective": '{"MetricName":"Accuracy"}'
+            }}
+    }
+
+    for _, params in problem_types_dict.items():
```

Comment:
better use parameterized tests here, or have two test which call the run_test with different parameter, because with for loop you run two tests in on test and if the first fails the second doesn't run